setup.py

```diff
@@ -2,16 +2,24 @@
 import codecs
 import os
 from setuptools import setup, find_packages
-try:
-    from pip.req import parse_requirements
-    from pip.download import PipSession
-except ImportError:
-    print('Cannot find pip.')
-    raise
-
-# Get requirements from the requirements.txt file.
-pip_requirements = parse_requirements("requirements.txt", session=PipSession())
-install_requires = [str(ir.req) for ir in pip_requirements]
+
+import pip
+if pip.__version__ < "10":
+    try:
+        from pip.req import parse_requirements
+        from pip.download import PipSession
+    except ImportError:
```

Comment:
I think the try/except for the import should be just next to "import pip" rather than here.